_drafts/page_all_frontmatter.md

```diff
@@ -42,18 +42,16 @@ sidebar: left
 comments: true
 breadcrumb: true
 show_meta: false
-meta_title:             # SEO: Overwrites title in <head> if needed
-meta_description:
```

Comment:
This stuff was removed to keep SEO meta more predictable and easier to manage using existing page description and title.